src/modules/Providers/views/details/tabs/Hosts/ProviderHosts.tsx

```diff
@@ -16,12 +16,14 @@ export type ProviderHostsProps = {
   loadError?: unknown;
 };
 
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
why leaving it like that? and not removing the unneeded component?

Reply:
Since it's the old code. I just fixed it to pass the eslint errors till refactoring the Hosts tab.